src/pages/workspace/reimburse/WorkspaceReimburseNoVBAView.js

```diff
@@ -56,17 +57,23 @@ class WorkspaceReimburseNoVBAView extends React.Component {
         },
     ];
 
-    /**
-     * Set the rate throttled by 3 seconds so the user does not have to type over the corrected value
-     */
-    updateRateValueThrottled = _.throttle((value) => {
-        this.setState({rateValue: this.getRateDisplayValue(value)});
+    updateRateValueDebounced = _.debounce((value) => {
```

Comment:
Let's do some cleanup first. 
1. Create a method updateRateValue which does the whole thing. 
2. then bind the bounced method on the constructor. 
```
this.updateRateValueDebounced = _.debounce(this.updateRateValue.bind(this), 1000);
```

3. Then move the unitItems property to constructor. 
4. constructor should be first method of the class.